### ISSUE TITLE
Help with Setup Tautulli Enhanced App

### ISSUE THREAD

**Issue by itr6** [linuxserver/Heimdall#305]
I cannot get my docker install to setup the Tautulli enhanced app. The link works fine to open up Tautulli but that's it. My Pihole enhanced app works great. I have tried setting the secondary URL in the config area to the direct URL and port instead of my Caddy Reverse proxy address. I have even tried setting the secondary URL to the IP address and port. It does not work. I have also tried changing API keys, turning API off and back on again and still nothing.

Can anyone provide any assistance? Is there something within Tautulli I need to setup for this to work?

Thanks.

**Comment by aptalca** [comment]
Likely a reverse proxy issue

Do you have authentication through the reverse proxy? If so turn it off for the api calls

**Comment by itr6** [comment]
Sorry about the duplicate issue. I thought that was a different repo.

I'm guessing it was a proxy issue. I deleted tautulli app and re-created it use the direct url and port and it worked. So I deleted that one and created a 3rd app with the app pointed to the reverse proxy and the API config URL pointing to the direct URL and now it works. When I had the original app, nothing API Config URL would work.
